src/schemas/json/typst.json

```diff
@@ -0,0 +1,245 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "https://json.schemastore.org/typst.json",
+  "title": "Typst Package Manifest",
+  "description": "Schema for typst.toml package manifest files, used by the Typst package manager and universe.",
+  "type": "object",
+  "required": ["package"],
+  "properties": {
+    "package": {
+      "type": "object",
+      "description": "Contains metadata about the package.",
+      "required": [
+        "name",
+        "version",
+        "entrypoint",
+        "authors",
+        "license",
+        "description"
+      ],
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "The package's identifier in its namespace. Should use kebab-case if it contains multiple words. Should not contain 'typst'.",
+          "pattern": "^[a-z0-9][a-z0-9\\-]*$",
+          "examples": ["my-package", "example", "uni-stuttgart-template"]
+        },
+        "version": {
+          "type": "string",
+          "description": "The package's version as a full major-minor-patch triple (following SemVer). See https://semver.org/",
+          "pattern": "^(0|[1-9]\\d*)\\.(0|[1-9]\\d*)\\.(0|[1-9]\\d*)(?:-((?:0|[1-9]\\d*|\\d*[a-zA-Z-][0-9a-zA-Z-]*)(?:\\.(?:0|[1-9]\\d*|\\d*[a-zA-Z-][0-9a-zA-Z-]*))*))?(?:\\+([0-9a-zA-Z-]+(?:\\.[0-9a-zA-Z-]+)*))?$",
+          "examples": ["1.0.0", "0.2.1"]
+        },
+        "entrypoint": {
+          "type": "string",
+          "description": "The path to the main Typst file (relative to the package root) that is evaluated when the package is imported.",
+          "examples": ["lib.typ", "src/main.typ"]
+        },
+        "authors": {
+          "type": "array",
+          "description": "A list of the package's authors. Each author can provide an email address (<mailto:EMAIL>), homepage (<https://URL>), or GitHub handle (<@HANDLE>) in angle brackets.",
```

Comment:
Email addresses should not be prefixed by `mailto`, you should just write it between angle brackets.